src/storage/state.rs

```diff
@@ -0,0 +1,96 @@
+use std::collections::HashMap;
+use serde::{Deserialize, Serialize};
+
+use crate::events::Linkdef;
+
+use derive_builder::Builder;
+
+////////////
+// Claims //
+////////////
+
+/// A Claim that can be applied to a given lattice
+#[derive(Debug,Serialize, Deserialize, Clone)]
+pub struct Claim {
+    pub name: String,
+
+    pub version: String, /// TODO(QUESTION): Do we enforce semver?
```

Comment:
To answer your question: No, this is just a usize

Reply:
I thought the `revision` was  a `usize` and the `version` is a string? That's how it is in [`events/data.rs`](https://github.com/wasmCloud/wadm/blob/wadm_0.4/src/events/data.rs#L49)

Reply:
Oh sorry you're correct. I was looking at this on phone and misread. To answer the todo question, we do expect this field to be semver

Reply:
Thanks, will change this to a `semver::Version` then!